### DISCUSSION TITLE
Icon states/colors

### DISCUSSION THREAD

**Discussion by mangus580** [cgiesche/streamdeck-homeassistant#322] [General]
I use different icons and colors to show different states for some things.

For example - I have an indicator on my dashboard for showing if the washing machine is idle, washing, or finished.

I also color the icons different for each status.

Is there a way to use the same setup for streamdeck?  It would be nice to be able to glance over and see the status on the streamdeck based more on color than the text.

Ideas?

**Comment by fkthisusrrequirement** [comment]
Yes! I just posted a solution on Home Assistant forums.

Here: https://community.home-assistant.io/t/changing-icon-color-based-on-situation/834869
